jdaviz/configs/mosviz/helper.py

```diff
@@ -22,8 +85,9 @@ def load_1d_spectra(self, data_obj, data_labels=None):
             ``data_obj``. Can be a list of strings representing data labels
             for each item in ``data_obj`` if  ``data_obj`` is a list.
         """
-        self.app.load_data(data_obj, parser_reference="mosviz-spec1d-parser",
-                           data_labels=data_labels)
+        self.app.load_data(
+            data_obj, parser_reference="mosviz-spec1d-parser", data_labels=data_labels
+        )
```

Comment:
```suggestion
        self.app.load_data(data_obj, parser_reference="mosviz-spec1d-parser",
                           data_labels=data_labels)
```

The original format was one less line and kept the line under 79 characters.